pkg/controllers/reconcilers.go

```diff
@@ -67,8 +87,18 @@ type ParentReconciler struct {
 	Config
 }
 
+// NewParentReconciler returns a ParentReconciler given an empty object to represent the parent's type, a Config,
+// and a list of SubReconcilers.
+func NewParentReconciler(objType runtime.Object, cfg Config, subReconcilers ...SubReconciler) *ParentReconciler {
```

Comment:
I'm 👎 on constructors that mask the semantic value of field names for positional args without adding meaningful behavior.

Reply:
Ok, I've inlined the constructor for now. We can always introduce a constructor or builder later if necessary.